sources/packages/backend/apps/db-migrations/src/sql/ProgramYear/Add-max-lifetime-bc-loan-amount.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE
+  sims.program_years
+ADD
+  COLUMN max_lifetime_bc_loan_amount NUMERIC(8, 2) NOT NULL DEFAULT 0;
```

Comment:
Just a suggestion, the DEFAULT can be set as 50000 and we can avoid lines 8 to 13.